### ISSUE TITLE
Similar apps question

### ISSUE THREAD

**Issue by remco555** [JoMingyu/google-play-scraper#126]
Hi, I am using the similar_apps field om the "app" function for a project I am doing.
I had noticed Google play changed and subsequently my project doesn't find apps anymore. After going through this repo I found that you removed this "similar_apps" field, among other. I am wondering why this is and if there are any plans to make this functional again?

**Comment by JoMingyu** [comment]
Due to the nature of Google Play DOM (css uglify), it must be crawled with the data initializer in the head tag. However, data related to the 'similar app' disappeared from there, so I had no choice but to remove the corresponding field from the library.

I will update it if it can be used again due to a change in the data structure of google play in the future. I always try to fetch as much data as possible.

**Comment by siftnoorsingh** [comment]
Agreed. This is a feature that I was looking for as well and hoping to raise a request for.

**Comment by JoMingyu** [comment]
It is developing at https://github.com/JoMingyu/google-play-scraper/pull/134. Greeting for new maintainer 🙏

**Comment by mv627** [comment]
Hi, I was wondering if the similar_apps feature had been re-implemented? I'm very very new to github so I wasn't sure where to find this info. Thank you!
